click-to-reveal/script.js

```diff
@@ -0,0 +1,56 @@
+const questions = [
+    {
+        'question': 'Have you heard about the chocolate record player? ',
+        'answer': 'It sounds pretty sweet.'
+    },
+    {
+        'question': 'What do you call a factory that makes okay products?',
+        'answer': 'A satisfactory.'
+    },
+    {
+        'question': 'What did one wall say to the other?',
+        'answer': "I'll meet you at the corner."
+    }
+]
+
+window.addEventListener("DOMContentLoaded", (event) => {
+    console.log("DOM fully loaded and parsed");
+
+    // step 1 get all the buttons 
+    // step 2 loop through all the buttons and add event listeners 
+
+    const buttons = document.getElementsByTagName('button');
+    console.log(buttons);
+
+    let y = 99
+
+    for (let x = 0; x < buttons.length; x++) {
```

Comment:
Since we are trying to load the data from your questions array, I think it may make more sense to remove all of the `<button>`s in your HTML, and instead generate and insert them here. By generating the buttons, you will not have to double check to make sure that the button exists with your `y` at all.

Reply:
Changed this to 

`let y = buttons.length + 1` 

so that it matches the amount of buttons instead if hard coding a number